### ISSUE TITLE
ToolkitSampleMultipleChoiceOption broken

### ISSUE THREAD

**Issue by Avid29** [CommunityToolkit/Labs-Windows#376]
### Describe the bug

The ``ToolkitSampleMultiChoiceOption`` is unable to parse data.

The error
![image](https://user-images.githubusercontent.com/9384045/221234258-d3a7a19c-1b88-4439-94ad-6f4b635fcb97.png)

The sample choices
![image](https://user-images.githubusercontent.com/9384045/221234329-84d916de-759e-4e79-8a54-ad9908e4ed84.png)

The sample XAML
![image](https://user-images.githubusercontent.com/9384045/221234590-a05e7b52-767a-41e9-ba27-4d4c1f8978c7.png)

### Steps to reproduce

```text
Use https://github.com/Avid29/Labs-Windows/tree/marquee
```

### Expected behavior

The strings should be parsed for XAML binding

### Code Platform

- [ ] UWP
- [ ] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Other (specify)

### Additional context

This is breaking my PR for #275

### Help us help you

No, I'm unable to contribute a solution.

**Comment by niels9001** [comment]
@Arlodotexe @Avid29 https://github.com/CommunityToolkit/Labs-Windows/pull/336 introduced changes in this space.

@Avid29 We now have the NumericOption (see example in the PR above) and is more appropriate. Could you try that to see if that works?

**Comment by Avid29** [comment]
@niels9001 I will try it for speed, however the ``RepeatBehavior`` is still broken and requires the Multi-Choice

**Comment by niels9001** [comment]
> @niels9001 I will try it for speed, however the ``RepeatBehavior`` is still broken and requires the Multi-Choice

See https://github.com/CommunityToolkit/Labs-Windows/blob/main/components/SegmentedControl/samples/SegmentedControlBasicSample.xaml.cs#L12

To parse strings to enum a concerting method is required. See the XAML/CS in the above file as a sample.

**Comment by Avid29** [comment]
[RepeatBehavior](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.animation.repeatbehavior?view=winrt-22621) is not an enum

**Comment by niels9001** [comment]
> [RepeatBehavior](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.animation.repeatbehavior?view=winrt-22621) is not an enum

Yeah, my point was that you might need to convert it manually.. (from string to x). See the previous example.

Might be the same for the speed stuff.. could be that it's parsed as a string, but your control asks for a double. Adding a function to convert should solve that (or using the NumericOption which is more appropriate)

**Comment by michael-hawker** [comment]
This seems related to this feature request I had added I think? https://github.com/CommunityToolkit/Labs-Windows/issues/149
